packages/docs-site/docs-consolidator.js

```diff
@@ -122,21 +122,39 @@ const packageLoop = () => {
 
 const allComponents = packageLoop()
 
+// Finds the current component within the matchDocs function
+const currentComponent = (thisFile) => allComponents.filter(component => (component.component.toLowerCase() === matter(fs.readFileSync(thisFile)).data.title.toLowerCase()) ? component : false
+)
+
+// Builds an output to display in the console when a doc has been processed
+const logOutputBuilder = (prefix, doc) => {
+  const logOutput = (prefix) ? `${prefix}/${doc}` : doc
+  return console.log(chalk.green(` ✓ Proccessing of ${chalk.blue(logOutput)}, complete.`))
+}
+
+// Proccesses all files found in the folderloop and if they are a markdown file they proccess them, 
+// if they are a directory then run folderloop again inside that directory.
+const matchDocs = (docsPath, doc, prefix) => {
+  const thisFile = join(docsPath, doc)
+  if (doc.match(/.md/gi)) {
+    injectInFile(thisFile, currentComponent(thisFile))
+    logOutputBuilder(prefix, doc)
+  } else {
```

Comment:
Could we get rid of the extra nesting here? Prettier might sort this out by itself.